src/main/java/Main.java

```diff
@@ -0,0 +1,14 @@
+import java.io.File;
+
+public class Main {
+
+    public static void main(String[] args) {
+        if (args.length == 2 && args[0].equals("-fileName")) {
```

Comment:
Is it's a file name or file path?

Reply:
a path that has a name 🤣 
renamed to filePath)